python/ray/util/dask/scheduler.py

```diff
@@ -361,27 +362,28 @@ def _rayify_task(
                 if alternate_return is not None:
                     return alternate_return
 
-        func, args = task[0], task[1:]
-        if func is multiple_return_get:
-            return _execute_task(task, deps)
-        # If the function's arguments contain nested object references, we must
-        # unpack said object references into a flat set of arguments so that
-        # Ray properly tracks the object dependencies between Ray tasks.
-        arg_object_refs, repack = unpack_object_refs(args, deps)
-        # Submit the task using a wrapper function.
```

Comment:
With the new Task format, deps seem to include results from other tasks. Rather than unpacking/repacking, we can just pass task and deps to a remote function.
This might be doing this anti-pattern https://docs.ray.io/en/latest/ray-core/patterns/pass-large-arg-by-value.html, so let me check.